test/include/materials/ADCheckGlobalToDerivativeMap.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ADMaterial.h"
+
+/**
+ * A material that couples a material property
+ */
```

Comment:
Not a very useful doc comment.

Reply:
Changed to something a bit more meaningful